libraries/testkit/cluster.py

```diff
@@ -301,6 +301,20 @@ def verify_alive(self, mode):
 
         return errors
 
+    def stop_sg_and_accel(self):
```

Comment:
This will try to stop SGAccel even if mode=="cc". In keywords/SyncGateway.py, we have a method stop_sync_gateways(self, cluster_config, url=None) which can either stop 1 SG or all SGs. We can add another similar method for SGAccel so they are will be in the same place.

Reply:
used the method from libraries.testkit.syncgateway and libraries.testkit.sgaccel stop methods. Can't use keyworks/SyncGateway stop as the same class 'SyncGateway' exists on both keywords/SyncGateway and libraries.testkit.syncgateway and it creates conflicts on cluster.py, so I used stop methods from libraries.testkit